bots/task/github.py

```diff
@@ -79,7 +79,7 @@ def determine_github_base():
                 stdout=subprocess.PIPE, cwd=BASE
             ).communicate()[0].strip()
         for f in formats:
-            m = f.match(remote_output)
+            m = f.match(remote_output.decode('utf-8'))
```

Comment:
FYI, `utf-8` is the default. No harm in mentioning it explicitly of course, just for future typing laziness.  :-)